.changeset/rich-fans-notice.md

```diff
@@ -0,0 +1,7 @@
+---
+'skuba': minor
+---
+
+deps: eslint-config-skuba 3
+
+This major upgrade brings in new rules from [typescript-eslint v6](https://typescript-eslint.io/blog/announcing-typescript-eslint-v6/).
```

Comment:
Do we expect existing projects to disable certain rules to begin with, and if so, can we provide an example `.eslintrc.js` diff?